---a/net/minecraft/dispenser/IBehaviorDispenseItem.java
+++b/net/minecraft/dispenser/IBehaviorDispenseItem.java
@@ -2,19 +2,19 @@
 
 import net.minecraft.item.ItemStack;
 
-public interface IBehaviorDispenseItem
-{
-    IBehaviorDispenseItem itemDispenseBehaviorProvider = new IBehaviorDispenseItem()
-    {
+public interface IBehaviorDispenseItem {
+
+    IBehaviorDispenseItem itemDispenseBehaviorProvider = new IBehaviorDispenseItem() {
+
         private static final String __OBFID = "CL_00001200";
-        public ItemStack dispense(IBlockSource source, ItemStack stack)
-        {
+
+        public ItemStack dispense(IBlockSource source, ItemStack stack) {
             return stack;
         }
     };
 
     ItemStack dispense(IBlockSource source, ItemStack stack);
-}
+}
